.circleci/config.yml

```diff
@@ -275,6 +303,73 @@ jobs:
       - store_test_results:
           path: libs/SalesforceHybrid/build/outputs/androidTest-results/
 
+  generate-artifacts:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache: *restore-gradle-cache
+      - restore_cache: *restore-node-cache
+      - restore_cache: *restore-ruby-cache
+      - run: *setup-env
+      - save_cache: *save-node-cache
+      - save_cache: *save-ruby-cache
+      - run: *compile-libraries
+      - store_artifacts:
+          path: libs/SalesforceAnalytics/build/outputs/aar/
+          destination: Libraries
+      - store_artifacts:
+          path: libs/SalesforceSDK/build/outputs/aar/
+          destination: Libraries
+      - store_artifacts:
+          path: libs/SmartStore/build/outputs/aar/
+          destination: Libraries
+      - store_artifacts:
+          path: libs/SmartSync/build/outputs/aar/
+          destination: Libraries
+      - store_artifacts:
+          path: libs/SalesforceHybrid/build/outputs/aar/
+          destination: Libraries
+      - store_artifacts:
+          path: libs/SalesforceReact/build/outputs/aar/
+          destination: Libraries
+      - run:
+          name: Build Native Sample Apps
+          when: always
+          command:  |
+            ./gradlew :native:NativeSampleApps:AppConfigurator:assemble
```

Comment:
`assembleDebug`?

Reply:
`assemble` by itself runs `assembleDebug` and `assembleRelease` right?

Reply:
Yup - I guess it doesn't hurt to archive the release artifacts as well.